tests/frontend/cypress/integration/blob.js

```diff
@@ -48,7 +50,12 @@ describe("Blob view test - Malwarecage", function () {
     cy.contains("inject");
     cy.contains("First seen");
     cy.contains("Last seen");
-
+    
+    cy.wait(5)
```

Comment:
Why do we need to wait? Test works without that.

PS.: Waiting x seconds is never a good idea.

Reply:
I know this is not very good idea but sometimes inevitable. Test works - sometimes, sometimes not, depends on env as well. It depends how fast elements are loaded in DOM. If they are loaded too fast, they will be not attached causing click command to fail. Unfortunately, retry was not implemented by Cypress in click command for some reasons...

Reply:
The battle was lost but we won the war. The solution is to wait for XHR after clicking on Details tab of blob :)